radix-engine-interface/src/api/key_value_store_api.rs

```diff
@@ -9,6 +9,9 @@ use sbor::rust::prelude::*;
 use sbor::LocalTypeId;
 use sbor::{generate_full_schema, TypeAggregator};
 
+pub const KV_STORE_DATA_SCHEMA_VARIANT_LOCAL: u8 = 0;
+pub const KV_STORE_DATA_SCHEMA_VARIANT_REMOTE: u8 = 1;
```

Comment:
Dead code?

Reply:
`KV_STORE_DATA_SCHEMA_VARIANT_LOCAL` is and the other is not.

It kind of reminds me that I can optimise the local variant as well.

So, both will be used.